modules/rosa-additional-principals-create.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-accessing-cluster.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-accessing-cluster.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-additional-principals-create_{context}"]
+= Adding additional principals while creating your ROSA cluster
+
+Use the `--additional-allowed-principals` argument to permit access through other accounts and roles.
+
+.Procedure
+
+. In your terminal, add `--additional-allowed-principals` to the `rosa create cluster` command, similar to the following:
++
+[source,terminal]
+----
+$ rosa create cluster [...] --additional-allowed-principals <arn_string>
+----
++
+You can use `arn:aws:iam::account_id:root` to approve an account, `arn:aws:iam::account_id:role/role_name` to a approve a specific role, `arn:aws:iam::account_id:user/user_name` to approve an additional user, and `arn:aws:iam::account_id:federated-user/user_name` to approve a federated user.
```

Comment:
Again - I think we only support /role so kindly check.

Reply:
When I tested it, it attached with a user role, though I wasn't sure how to test beyond that. Do you know who would be our QE contact for this? Maybe I can get access to their test plan.

Reply:
@bchandra-ocp confirmed it is just user roles. I've updated the docs.

`W: The current version (1.2.41-RC1) is not up to date with latest released version (1.2.41).`
`W: It is recommended that you update to the latest version.`
`E: Failed to update cluster: Principal 'arn:aws:iam::123456789012:root' must be of format:`
`'arn:aws:iam::123456789012:role/name'`